src/logger/loggerApi.c

```diff
@@ -275,18 +275,26 @@ static void get_bt_status(struct Serial* serial, const bool more)
 #if BLUETOOTH_SUPPORT
         json_objStartString(serial, "bt");
         json_int(serial, "init", (int)bt_get_status(), 0);
-        json_objEnd(serial, 1);
+        json_objEnd(serial, more);
 #endif
 }
 
 static void get_logging_status(struct Serial* serial, const bool more)
 {
-#if SDCARD_SUPPORT
         json_objStartString(serial, "logging");
         json_int(serial, "status", (int)logging_get_status(), 1);
         json_int(serial, "dur", logging_active_time(), 0);
-        json_objEnd(serial, 1);
-#endif
+        json_objEnd(serial, more);
+}
+
+int api_send_logging_status(struct Serial* serial)
```

Comment:
This sends just the logging status in JSON form to the connected clients.

We have another API method that sends multiple statuses